.github/renovate.json5

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi"
+  ],
+  "packageRules": [
+    {
+      "datasources": ["docker"],
+      "updateTypes": ["patch"],
+      "groupName": "docker patch updates"
+    },
+    {
+      "datasources": ["go"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "go.mod non-major updates"
+    },
```

Comment:
```suggestion
    {
      "description": "Don't separate Go minor/patch as will be grouped together anyway",
      "datasources": ["go"],
      "separateMinorPatch": false
    }, {
      "description": "Group together all non-major Go updates",
      "datasources": ["go"],
      "updateTypes": ["patch", "minor"],
      "groupName": "go.mod non-major updates"
    },
```

Reply:
I believe @DavidSpek intentionally wanted to separate minor and patch updates, which is also what I want, because at the start, I'd like to merge some patch updates first to reduce risk.

Do you have any suggestion the best config in this case?

Reply:
Yes, in that case I can suggest something different

Reply:
Actually I think how it is now in the config is right - let's wait for the next refresh to confirm